tests/suites/tenant/diagnostics/Diagnostics.ts

```diff
@@ -0,0 +1,75 @@
+import type {Locator, Page} from '@playwright/test';
+
+import {VISIBILITY_TIMEOUT} from '../TenantPage';
+
+export enum DiagnosticsTab {
+    Info = 'Info',
+    Schema = 'Schema',
+    TopShards = 'Top shards',
+    Nodes = 'Nodes',
+    Graph = 'Graph',
+    Tablets = 'Tablets',
+    HotKeys = 'Hot keys',
+    Describe = 'Describe',
+}
+
+export class Diagnostics {
+    private tabs: Locator;
+    private schemaViewer: Locator;
+    private dataTable: Locator;
+    private primaryKeys: Locator;
+    private refreshButton: Locator;
+    private autoRefreshSelect: Locator;
+
+    constructor(page: Page) {
+        this.tabs = page.locator('.kv-tenant-diagnostics__tabs');
+        this.schemaViewer = page.locator('.schema-viewer__table');
```

Comment:
It's datatable as well

Reply:
Not sure if understood properly
You mean to rename schemaViewer -> schemaViewerDatatable ?

Reply:
No, I mean you have `this.dataTable` selector, why do you need additional selector for `SchemaViewer` `DataTable`?